src/views/router_views/individualPscList/individualPscList.njk

```diff
@@ -0,0 +1,43 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% set title = i18n.individual_psc_list_main_title %}
```

Comment:
You can just use the i18n.individual_psc_list_main_title directly in the code you don't need to set a variable here first so you can remove this.

Reply:
as I only learnt this morning I think we do need this line, in order for the localised value from the english/welsh files to be passed into the `partials/__meta_header` template

Reply:
@clewis1 this line needs to be present in the template in order for the Tab to display the page title

Reply:
👍